openconfig_pyang/plugins/openconfig.py

```diff
@@ -21,7 +21,7 @@
 
 from __future__ import print_function
 
-from enum import Enum
+from enum import IntEnum, Enum
```

Comment:
is `enum` a standard library package?   it tripped me up when using on new systems

Reply:
Updated to enum34, it's not a standard package. It's listed in `requirements.txt`.